.github/CODEOWNERS

```diff
@@ -84,6 +84,17 @@
 
 /code/modules/wiremod/ @Watermelon914
 
+# Ryll-Ryll
+
+/code/__DEFINES/wounds.dm @Ryll-Ryll
+/code/modules/jobs/job_types/chief_medical_officer.dm @Ryll-Ryll
+/code/modules/jobs/job_types/medical_doctor.dm @Ryll-Ryll
+/code/modules/jobs/job_types/paramedic.dm @Ryll-Ryll
+/code/modules/surgery/ @Ryll-Ryll
```

Comment:
You're gonna have to merge yours and Cobby's things together and put it in "Multiple Owners". At least I think you have to, the others are like that